src/app/clientPage.tsx

```diff
@@ -100,22 +103,27 @@ export default function ClientPage({
   return (
     <>
       {displayResult ? (
-        <Result
-          trackShare={trackShare}
-          summaryResponse={currentResult as TextSummaryResponseType | SongMeaningResponseType}
-          handleNewSearchBtnClick={handleNewSearchBtnClick}
-          originalContent={originalContent}
-          songDetails={songDetails}
-          isLoadingSSE={isLoadingSSE}
-        />
+        <MinHeightBodyContainer>
+          <Result
+            trackShare={trackShare}
+            summaryResponse={currentResult as TextSummaryResponseType | SongMeaningResponseType}
+            handleNewSearchBtnClick={handleNewSearchBtnClick}
+            originalContent={originalContent}
+            songDetails={songDetails}
+            isLoadingSSE={isLoadingSSE}
+          />
+        </MinHeightBodyContainer>
       ) : (
         <>
-          <InputPageHeader />
-          <InputComponent
-            handleFormSubmit={handleFormSubmit}
-            onInputChange={trackInputSelection}
-            onLengthChange={trackLengthSelection}
-          />
+          <MinHeightBodyContainer>
+            <InputPageHeader />
+            <InputComponent
+              handleFormSubmit={handleFormSubmit}
+              onInputChange={trackInputSelection}
+              onLengthChange={trackLengthSelection}
+            />
+          </MinHeightBodyContainer>
+          <About />
```

Comment:
[this will get pre-rendered from the server](https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components)